vision/vision_general/scripts/receptionist_commands.py

```diff
@@ -34,15 +35,16 @@
 CAMERA_TOPIC = "/zed/zed_node/rgb/image_rect_color"
 CHECK_PERSON_TOPIC = "/vision/detect_person"
 PERSON_DESCRIPTION_TOPIC = "/vision/person_description"
+PERSON_POSTURE_TOPIC = "/vision/person_posture"
 BEVERAGE_TOPIC = "/vision/beverage_location"
 FIND_SEAT_TOPIC = "/vision/find_seat"
 IMAGE_TOPIC = "/vision/img_person_detecion"
```

Comment:
not for the scope of this pr, but these should be in frida_constans @Ale-Coeto ...